account/src/main/java/com/whz/account/resource/AccountResource.java

```diff
@@ -23,24 +23,54 @@
 import com.whz.account.infrastructure.persistence.AccountQueries;
 import com.whz.account.infrastructure.persistence.QueryModelStateStoreProvider;
 import com.whz.account.model.account.Account;
-import com.whz.account.model.account.AccountEntity;
+import io.vlingo.actors.Logger;
 import io.vlingo.actors.Stage;
 import io.vlingo.common.Completes;
 import io.vlingo.http.Response;
 import io.vlingo.http.resource.Resource;
 import io.vlingo.http.resource.ResourceHandler;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
 
+/**
+ * The AccountResource is responsible for managing the REST interface and invoking methods/events.
+ *
+ * @author Lation
+ */
 public class AccountResource extends ResourceHandler {
 
+  private Logger logger;
+
   private final Stage stage;
   private final AccountQueries accountQueries;
+  private final String portfolioUrl = System.getenv("PORTFOLIO_URL");
 
+  /**
+   * Sets the stage and accountQueries and initializes a StockAcquiredSubscriber singleton.
+   *
+   * @param stage - stage object of the world
+   */
   public AccountResource(final Stage stage) {
+    StockAcquiredSubscriber saSub = StockAcquiredSubscriber.INSTANCE;
+    saSub.setStage(stage);
```

Comment:
You could change it because you don't need this variable anywhere.
```suggestion
    StockAcquiredSubscriber.INSTANCE.setStage(stage);
```